tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "baseUrl": "./src",
     "rootDir": "./",
-    "target": "es2016",
+    "target": "es2017",
```

Comment:
Is this a breaking change?

Reply:
Not a breaking change. Per the commit message:

Prior to this change TS would always output polyfills for es2017 features.
By targeting es2017 TS will leave es2017 features untouched and then
babel shall use babel-preset-env to polyfill them based upon our browser
support (as defined in browserslist in package.json)

This means that eventually if our browerslist gets to the point where everything we support doesn't need a es2017 polyfill (e.g. Object.values()) then we won't output a polyfill for it at all.